package.json

```diff
@@ -29,6 +29,8 @@
   ],
   "devDependencies": {
     "babel-preset-es2015": "^6.18.0",
+    "eslint-config-incuna": "1.0.0",
+    "eslint-plugin-import": "2.2.0",
```

Comment:
Not so sure about this, as it would mean that updates would mean breaking builds. Not sure what would non-breaking changes to the config file could be, as all of the rule changes would require an update in the code